### ISSUE TITLE
It's difficult to find a direct link to a download

### ISSUE THREAD

**Issue by dblock** [opensearch-project/project-website#251]
There's a beautiful dropdown to select a platform to download, but what I need is a direct link so I can `wget` the .tar.gz from a remote system. Can we please either change the beautiful dropdown to a list or add direct links?

<img width="871" alt="Screen Shot 2021-08-05 at 5 19 59 PM" src="https://user-images.githubusercontent.com/542335/128422698-e7895d9e-443f-4497-811b-ed26361022ee.png">

**Comment by stockholmux** [comment]
@dblock Yep - I get the use-case. I've been thinking about something like this:

<img width="302" alt="Screen Shot 2021-08-05 at 3 21 17 PM" src="https://user-images.githubusercontent.com/1152927/128423017-4b748b3f-4b41-4f95-8674-6a5a6ba0087d.png">

Or even a "Copy as cURL" button.

Thoughts?

**Comment by stockholmux** [comment]
[PS - those are links-styled-as-buttons, so you can right click "Copy as Link" right now]

**Comment by dblock** [comment]
I like both copy link or copy as cURL (as `wget` though?)!

**Comment by vimalMK** [comment]
@dblock @stockholmux  Interesting proposal and I think it will add some value to use experience. If the below looks good, I can commit a PR for this issue. 

![image](https://user-images.githubusercontent.com/35750792/199051973-10261ccd-ee97-46d7-83b3-58b65710c9a5.png)

**Comment by dtaivpp** [comment]
@vimalMK I think a copy link button would be great! We had gotten some other feedback that is probably going to have us redesign the entire page shortly. What do you think about this? In particular do you like the linked sample sites? 

https://github.com/opensearch-project/project-website/issues/862#issuecomment-1171680400

**Comment by vimalMK** [comment]
@dtaivpp In my opinion , I really like keeping the user in a single page while going through Downloads. That being said keeping the redirection minimal.  Sometimes that's not always the case for example the docker and ECR in this case and I like how it is right now. But if the redesign is discussed I like [Kafka](https://kafka.apache.org/downloads) downloads page a lot!! . Its like a one stop solution. . Just my thoughts . :)
